lblod/frontend-form-builder
# SFB-199: new builder SKELETON

## ID
 [SFB-199](https://binnenland.atlassian.net/browse/SFB-199)

 ## Description

Create the skeleton on how the builder will look like. This means a sidebar and the segments we want to create are visible when going to route `/formbuilder/:id/editor`

 ## Type of change


 ## How to test

1. Click on a form that you want to edit
2. See if looks like the wire frame in the attachments

 ## Links to other PR's

 - /

## Attachments

![image](https://github.com/lblod/frontend-form-builder/assets/36266973/70191c6c-6858-4039-b8cb-dd620bff2f2c)
